TodoListWebApp/Startup.cs

```diff
@@ -45,12 +51,12 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 
             app.UseStaticFiles();
 
+            app.UseSession(); // Needs to be app.UseAuthentication() and app.UseMvc() otherwise you will get an exception "Session has not been configured for this application or request."
```

Comment:
The position of UseSession() relatively to UseAuthentication() and UseMvc() is important (it needs to be just before them).